### DISCUSSION TITLE
twitter应该直接用gae代理，否则等半天

### DISCUSSION THREAD

**Discussion by ottilis** [SeaHOH/GotoX#49] [点子]
.twitter.com
.twitpic.com
t.co
.tweetdeck.com
这几个自有ip全封了。

**Comment by SeaHOH** [comment]
可以自行修改添加规则，具体写法看配置文件注释和默认示例。

另，其实我不太看 twitter 的，如果觉得合适，可以 pr。

**Comment by ottilis** [comment]
好了，开始不知道规则加哪，结果少了twitter.com也不行。下面的位置。

[50-gae] #通过 GAE 服务器代理

.twitter.com
twitter.com
.twitpic.com
t.co
.tweetdeck.com

**Comment by SeaHOH** [comment]
嗯，能用就好。

最下面还有一个 gae 组，自己也可以增加其它组，还可以 改变作用顺序（改数字就行）。

**Comment by ottilis** [comment]
很好用，目前最可靠最持续的稳定。

**Comment by pgz123** [comment]
已经没法FQ

**Comment by SeaHOH** [comment]
可以扫扫看 https://github.com/myguybetter/GoogleIPHunter

**Comment by pgz123** [comment]
找不到

**Comment by ottilis** [comment]
是没有，我也扫完了。
用原版，启用quic，用gscan_quic_windows_amd64.exe扫描，有许多。

**Comment by SeaHOH** [comment]
那就没法了，我这个没法用 QUIC。

建议你们换 GoProxy 客户端用 QUIC。

**Comment by pgz123** [comment]
找不到文件

**Comment by SeaHOH** [comment]
什么文件啊？

**Comment by pgz123** [comment]
这个，https://github.com/myguybetter/GoogleIPHunter啥都没有

**Comment by SeaHOH** [comment]
切换分支 master

**Comment by pgz123** [comment]
谢谢

**Comment by ottilis** [comment]
下午又开放了一些传统ip。

**Comment by ottilis** [comment]
今天又全封了，现在一封就密不透风了，只有quic。

**Comment by pgz123** [comment]
海哥有啥工具可以把ip变成一行一个？

**Comment by pgz123** [comment]
放GotoX里面

**Comment by SeaHOH** [comment]
过几天写一个扫描工具，正好 XX 那边有验证过的 IP。

**Comment by pgz123** [comment]
谢谢，辛苦了

**Comment by pgz123** [comment]
23:20:33 W 173.194.24.83 create_ssl_connection 'https://www.youtube.com/' 失败：
timeout('timed out',)
23:20:34 W 173.194.24.83 _create_ssl_connection 'www.youtube.com' 返回 timeout('
timed out',)，重试
23:20:35 W L4:52951->219.76.4.4 转发到 'twitter.com' 失败：timeout('timed out',)

23:20:35 W 173.194.24.76 create_ssl_connection 'https://www.youtube.com/' 失败：
timeout('timed out',)
23:20:35 W L4:52890-> do_DIRECT "GET https://www.youtube.com/" 失败，返回 404
23:20:36 W L4:52954->219.76.4.4 转发到 'twitter.com' 失败：timeout('timed out',)

23:20:43 W 219.76.4.4 _create_connection 'twitter.com' 返回 timeout('timed out',
)，重试
23:20:44 W 219.76.4.4 _create_connection 'twitter.com' 返回 timeout('timed out',
)，重试
23:20:51 W L4:52951->219.76.4.4 转发到 'twitter.com' 失败：timeout('timed out',)

23:20:58 W L4:52951-> do_FORWARD 链接远程主机 ('twitter.com', 443) 失败，尝试使
用 "FAKECERT" 规则。
23:20:58 W L4:52954->219.76.4.4 转发到 'twitter.com' 失败：timeout('timed out',)

23:20:58 W L4:52954-> do_FORWARD 链接远程主机 ('twitter.com', 443) 失败，尝试使
用 "FAKECERT" 规则。
23:20:58 W 将 'https://twitter.com' 加入 "FAKECERT" 规则 15 分钟。
浏览器显示404 无法找到给定的网址

**Comment by SeaHOH** [comment]
自己动手改规则
